### ISSUE TITLE
update label of scalar measurement datum

### ISSUE THREAD

**Issue by zhengj2007** [information-artifact-ontology/IAO#13]
_From [mcour...@gmail.com](https://code.google.com/u/116795168307825520406/) on March 04, 2009 12:10:58_

Hi,

During the OBI workshop in Vancouver the OBI group identified some terms to
be added/updated in IAO. If there is no objection this term will be updated
in the IAO file on Tuesday March 10th.

Change label of scalar measurement datum to scalar measurement data item.

Melanie

_Original issue: http://code.google.com/p/information-artifact-ontology/issues/detail?id=13_

**Comment by zhengj2007** [comment]
_From [alanruttenberg@gmail.com](https://code.google.com/u/alanruttenberg@gmail.com/) on March 05, 2009 21:54:18_

Not personally comfortable with this term, but soliciting opinion from other developers. Worst case alternative or 
community specific term. Please check in before changing.

**Comment by zhengj2007** [comment]
_From [mcour...@gmail.com](https://code.google.com/u/116795168307825520406/) on March 05, 2009 22:35:16_

We changed the label of datum to data item, keeping datum as alternative term.
Changing scalar measurement datum (and probably measurement datum) seems appropriate.
What about updating and keeping the -datum form as alternative term?

**Comment by zhengj2007** [comment]
_From [alanruttenberg@gmail.com](https://code.google.com/u/alanruttenberg@gmail.com/) on March 06, 2009 00:12:26_

I don't think it necessarily follows that measurement datum should follow the parent.

**Comment by zhengj2007** [comment]
_From [james.ma...@gmail.com](https://code.google.com/u/105349588160612283402/) on March 06, 2009 01:40:27_

Alan, what are you not comfortable with, the the class itself or the label change
from datum to data item (the label change surely means the same thing as datum or
data item)?

JM

**Comment by zhengj2007** [comment]
_From [alanruttenberg@gmail.com](https://code.google.com/u/alanruttenberg@gmail.com/) on March 06, 2009 06:50:46_

label change

**Comment by zhengj2007** [comment]
_From [mcour...@gmail.com](https://code.google.com/u/116795168307825520406/) on March 11, 2009 15:28:24_

Proposed resolution following discussion Alan/Melanie Wednesday 11th March

Terms datum and data item were deemed synonyms as they were both alternative terms
for data item, however they are not exact synonyms, as datum refers specifically to
the singular form.

Data item:
remove alternative term datum from data item
adding editor note saying "2009-xx-xx: removed datum as alternative term as datum
specifically refers to singular form, and is thus not an exact synonym"
adding editor note saying "data item deliberatly ambiguous: we merged data set and
datum to be one entity, no knowing how to define singular versus plural. So data item
is more general than datum. "

scalar measurement datum:
we decided to keep datum singular in scalar measurement datum, as in this case we
explicitly refer to the singular form
